test/helpers.sh

```diff
@@ -24,11 +24,27 @@ function check_folder {
 
 function check_folder_icon {
   FILE="${1}/"
-  if test -f "${FILE}"Icon$'\r'
+  ICON="${FILE}"Icon$'\r'
+  # Check for "custom icon" attribute on target
+  if xattr -p com.apple.FinderInfo "${FILE}" > /dev/null
   then
-    echo "✅ Folder ${FILE} has an icon as expected."
+    # Check for "invisible" attribute on icon
```

Comment:
Thanks! Would you mind returning early when there is an error, to minimize nesting?

Reply:
@lgarron I reworked the function to avoid nested ifs - PTAL